themes/aframe/source/releases/latest/aframe.js

```diff
@@ -60277,3 +60277,4 @@ module.exports={
 },{}]},{},[28])(28)
 });
 //# sourceMappingURL=aframe.js.map
+console.warn('The URL `https://aframe.io/releases/latest/aframe.js` is deprecated and is permanently fixed to v0.1.3. It is recommended to either upgrade or fix to a specific version like `https://aframe.io/releases/0.1.3/aframe.js`');
```

Comment:
> So is it guaranteed `https://aframe.io/docs/guide/getting-started.html#Builds` will last forever?

that's what legacy redirects are for. AMO has redirects from 2009. it's so URLs doesn't break :+1: 

okay, if you want. figured it'd be wise to be helpful.

I would still recommending reword "fix to" - I know what you mean, but I don't think that's a [common phrase](https://www.google.com/search?q="fix+to a version")